CHANGELOG.md

```diff
@@ -61,7 +60,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Changed http code on create index API with bad input raising NotXContentException from 500 to 400 ([#4773](https://github.com/opensearch-project/OpenSearch/pull/4773))
 - Change http code for DecommissioningFailedException from 500 to 400 ([#5283](https://github.com/opensearch-project/OpenSearch/pull/5283))
 - Pre conditions check before updating weighted routing metadata ([#4955](https://github.com/opensearch-project/OpenSearch/pull/4955))
-- Remove --enable-preview feature flag since Apache Lucene now patches class files ([#5642](https://github.com/opensearch-project/OpenSearch/pull/5642))
+- Standardize snapshot indices parsing so that combinations of included and excluded indices are treated the same regardless of the order they are listed in ([#5626](https://github.com/opensearch-project/OpenSearch/pull/5626))
```

Comment:
The idea here is that a user can scan this list and quickly determine whether a change might impact them. It should also be as concise as possible, which can obviously be a challenge. At a minimum, it should list what changed from a user's perspective, which in this case I think is the snapshot restore and clone APIs. What do you think of:

"Fix index exclusion behavior in snapshot restore and clone APIs"

Users that don't use those APIs or don't use exclusions can quickly identify this change as not impacting them, otherwise they'll need to dig into the specifics of the linked issue to get more info.

Reply:
Also, I believe we should backport this change to 2.x, so the changelog entry should go in the "Unreleased 2.x" section.

Reply:
Gotcha. I had thought that you had requested that the CHANGELOG entry be swapped to the new description alongside the PR title. My misunderstanding. I agree that the new option is much easier to quickly read over. 

I will move it to the Unreleased section as well.